projects/storefrontlib/src/cms-components/storefinder/components/store-finder-search-result/store-finder-list/store-finder-list.component.ts

```diff
@@ -33,4 +41,13 @@ export class StoreFinderListComponent {
     const storeListItem = this.document.getElementById('item-' + index);
     storeListItem.scrollIntoView();
   }
+
+  showStoreDetails(location: PointOfService) {
+    this.isDetailsModeVisible = true;
+    this.storeDetails = location;
+  }
+
+  hideStoreDetails() {
+    this.isDetailsModeVisible = false;
```

Comment:
Do you think can we have any edge cases, where it will be better to set here storeDetails = null? (refresh delay can show wrong details for while).
Although I can't observe that, it worries me

Reply:
We don't need that, because we are not fetching data asynchronously from store, just we grab them directly from local variable which always be resolved and contains proper data. Here it's optional to set `null` every time.